docs/sdkx_android/getting-started.mdx

```diff
@@ -30,8 +30,7 @@ Android app right away.
 
 ## Requirements {#minimum-supported-android-version}
 
-The Helpshift SDK X for Android requires minimum API level to be 16.
-However, **all the Helpshift's public apis will be non-operable below android SDK version 21.**
+Customers who are on/moving to SDK X, need to ensure they are supporting installing their apps/games only on API Level 24 and above [Android OS 7 and above]. After 10.3 is released, they can start supporting API Level 21 and above.
```

Comment:
If you are on/moving to SDK X, you need to ensure that you are supporting installing your apps/games only on API Level 24 and above [Android OS 7 and above]. After SDK X v10.3.0 is released, you can start supporting API Level 21 and above.